src/TLE.ts

```diff
@@ -788,10 +794,11 @@ export class UndefinedVariablePropertyVisitor extends Visitor {
     }
 
     private addIssue(tlePropertyAccess: PropertyAccess): void {
-        const propertyName: string = tlePropertyAccess.nameToken.stringValue;
+        const propertyName: string = tlePropertyAccess.nameToken ? tlePropertyAccess.nameToken.stringValue : "(unknown)";
```

Comment:
nit: Can we put tlePropertyAccess.nameToken in a local since it's used in multiple places.